netweaver/ha_cluster.sls

```diff
@@ -89,6 +99,12 @@ adapt_sap_profile_ers_{{ instance_name }}:
     - pattern: '^Restart_Program_00 = local \$\(_ER\) pf=\$\(_PFL\) NR=\$\(SCSID\)'
     - repl: 'Start_Program_00 = local $(_ER) pf=$(_PFL) NR=$(SCSID)'
 
+remove_autostart_option_{{ instance_name }}:
+  file.line:
+    - name: {{ profile_file }}
+    - match: ^Autostart = 1.*$
```

Comment:
is the` $ ` at end part of regex for searching `Autostart = 1`

Reply:
Yes. It's true that it doesn't have many sense, as we have the `.*` before in any case. It means that no more characters should be after this pattern